mods/d2k/rules/defaults.yaml

```diff
@@ -222,7 +225,7 @@
 	Inherits@2: ^SpriteActor
 	AppearsOnRadar:
 		UseLocation: yes
-	HiddenUnderFog:
+	HiddenUnderShroud:
```

Comment:
I assumed that it was done to make them visible to their owner under the fog.  That should be done by removing the `AlwaysVisibleStances` line below instead.